crates/swc/src/builder.rs

```diff
@@ -177,6 +177,10 @@ impl<'a, 'b, P: swc_ecma_visit::Fold> PassBuilder<'a, 'b, P> {
             ))
         } else {
             Either::Right(chain!(
+                Optional::new(
+                    compat::experimental::experimental(),
```

Comment:
This should not be variant of `EsVersion`

Reply:
I meant `Experiemental` type declared in `swc`.

Reply:
You mean adding an option to `JscExperimental` to _keep_ import assertions? I think I was in the mindset that the transformation to strip them had to be just like any other compat transformation, and I missed the fact that this was an option.

Reply:
Also, I just noticed that there is already a transformation to strip import assertions in `swc_ecma_transforms_proposal`, although it doesn't handle exports.

Reply:
> You mean adding an option to JscExperimental to keep import assertions? 

Yes, it's still stage 3, so keeping it should be opt-in.
Also, it's experimental because it's stage 3 and will be moved to `EsVersion` in the future.